### DISCUSSION TITLE
Get 'ImportError' when starting start_ofd.py in Windows or Linux??

### DISCUSSION THREAD

**Discussion by hsg120** [UltimaHoarder/UltimaScraper#937] [Q&A]
***Apologies!! I thought I checked, but I see there's an issue already open regarding this issue. Please delete or disregard...

I normally use this program in Linux, and have successfully run it previously. I recently upgraded to Python 3.9 and now I am unable to get the program to start. 

I thought maybe I had an unmet dependency in Linux, so I tried installing in Windows and got the exact same error, word for word:

python .\start_ofd.py
Traceback (most recent call last):
  File "C:\Users\xxx\OnlyFans-6.4.2\start_ofd.py", line 10, in <module>
    main_test.check_config()
  File "C:\Users\xxx\OnlyFans-6.4.2\tests\main_test.py", line 23, in check_config
    import helpers.main_helper as main_helper
  File "C:\Users\xxx\OnlyFans-6.4.2\helpers\main_helper.py", line 5, in <module>
    from sqlalchemy.ext.declarative import api
ImportError: cannot import name 'api' from 'sqlalchemy.ext.declarative' (C:\Users\xxx\AppData\Local\Programs\Python\Python39\lib\site-packages\sqlalchemy\ext\declarative\__init__.py)

I have run pip install -r requirments.txt and did not receive any errors.

Any idea what I'm doing wrong? Thank you!

**Comment by UltimaHoarder** [comment]
Use latest release

**Reply by hsg120** [reply]
That did the trick, thank you my friend!

**Reply by g0dzcsgo** [reply]
Got same issue, latest release not solving it!

**Reply by hsg120** [reply]
> Got same issue, latest release not solving it!

Are you using Windows or Linux? I can confirm 6.4.2 works on both platforms. 

What error are you getting? The same one that I posted in OP?

**Reply by g0dzcsgo** [reply]
Yea im getting the same one, I am using ubuntu 18.04...

And yea the error is the same as years something about cant import api from sqlalchemy.declarative

And i saw your discussion on here and downloaded the newest version, but still the same issue.

**Reply by hsg120** [reply]
Hmmm, I'm sorry if I'm asking the obvious...but did you try re-running the pip install -r requirements.txt again after you downloaded the latest version? I believe I had the same issue, up until I re-ran that requirements.txt command.
